### PR TITLE
Amazon-AWS-EC2-Volumes2

### PR DESCRIPTION
**Pull request by mihaitusa**
Delete volume operation.
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@gituser173 Please review.

File: cs-jClouds/src/main/java/io/cloudslang/content/jclouds/actions/volumes/DeleteVolumeInRegionAction.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR)
+            }
+    )
+    public Map<String, String> execute(@Param(value = Inputs.CommonInputs.PROVIDER, required = true) String provider,
+                                       @Param(value = Inputs.CommonInputs.ENDPOINT, required = true) String endpoint,
+                                       @Param(Inputs.CommonInputs.IDENTITY) String identity,
+                                       @Param(value = Inputs.CommonInputs.CREDENTIAL, encrypted = true) String credential,
+                                       @Param(Inputs.CommonInputs.PROXY_HOST) String proxyHost,
+                                       @Param(Inputs.CommonInputs.PROXY_PORT) String proxyPort,
+
```
Comment (Bonczidai) [inline cs-jClouds/src/main/java/io/cloudslang/content/jclouds/actions/volumes/DeleteVolumeInRegionAction.java]: Y the newline

File: cs-jClouds/src/main/java/io/cloudslang/content/jclouds/services/helpers/AmazonVolumeServiceHelper.java [hunk 2]
```diff
@@ -17,8 +17,11 @@
     private static final String SC1 = "sc1";
 
     private static final int FOUR = 4;
+    private static final int ONE_HUNDRED = 100;
     private static final int FIVE_HUNDRED = 500;
     private static final int TEN_THOUSANDS = 10000;
+    private static final int COMMON_LARGE_VALUE = 16384;
+    private static final int TWENTY_THOUSANDS = 20000;
```
Comment (Bonczidai) [inline cs-jClouds/src/main/java/io/cloudslang/content/jclouds/services/helpers/AmazonVolumeServiceHelper.java]: without `s` ? : )

**Comment by Bonczidai** [comment]
:+1:

File: cs-jClouds/src/main/java/io/cloudslang/content/jclouds/actions/volumes/DeleteVolumeInRegionAction.java (hunk 1 above)
Comment (mihaitusa) [inline cs-jClouds/src/main/java/io/cloudslang/content/jclouds/actions/volumes/DeleteVolumeInRegionAction.java]: @Bonczidai  I know... just keep that line to make some delimitation between common inputs and specific inputs (to be more easier to follow)

File: cs-jClouds/src/main/java/io/cloudslang/content/jclouds/services/helpers/AmazonVolumeServiceHelper.java (hunk 2 above)
Comment (mihaitusa) [inline cs-jClouds/src/main/java/io/cloudslang/content/jclouds/services/helpers/AmazonVolumeServiceHelper.java]: @Bonczidai  You are right ... as usual :+1:  Fixed...

**Pull request merged**